src/pgp.rs

```diff
@@ -272,6 +272,14 @@ pub async fn pk_encrypt(
     .await
 }
 
+/// Decrypts the message with keys from the private key keyring.
+///
+/// Receiver private keys are provided in
+/// `private_keys_for_decryption`.
+///
+/// If `ret_signature_fingerprints` is not `None`, stores fingerprints
+/// of all keys form `public_keys_for_validation` keyring that have
```

Comment:
Not sure. How about something ala
>fingerprints of all keys from the -___- keyring are stored, if they have valid sigs?
```suggestion
/// of all keys from the `public_keys_for_validation` keyring that have
```